### DISCUSSION TITLE
Add information about architecture

### DISCUSSION THREAD

**Discussion by rbas** [stanislav-tkach/os_info#334] [Ideas]
Hi @stanislav-tkach, 
I would like to ask you whether you consider to add informations about architechture into os informations? 

_If yes I can obviously try to make PR, I just would like to ask before I'll put any efford :D_

Thank you

**Comment by stanislav-tkach** [comment]
Hello @rbas!
Sorry, I somehow missed this discussion, but I have merged your pull request and I'm going to release a new version soon.

**Comment by stanislav-tkach** [comment]
I have released the 3.7.0 version with this feature.
